src/components/common/Tabs/TabContent.tsx

```diff
@@ -6,15 +6,12 @@ type Props = {
   activeTab: number
 }
 
-const TabContent: React.FC<Props> = (props) => {
-  const { tabItems, activeTab } = props
-  return (
-    <div>
-      {tabItems.map((tab) => {
-        return tab.id === activeTab ? tab.content : null
-      })}
-    </div>
-  )
-}
+const TabContent: React.FC<Props> = ({ tabItems, activeTab }) => (
+  <div>
+    {tabItems.map((tab) => {
+      return tab.id === activeTab ? tab.content : null
+    })}
```

Comment:
`map` loops over the whole array, plus react has to render an array (even if it's mostly filled with nulls) in the end
```suggestion
    {tabItems.find((tab) => tab.id === activeTab)?.content}
```
exits early

Reply:
Can we commit this suggestion @W3stside ? Good to go otherwise.

Reply:
@Velenir @biocom I'm more concerned about why this is even mapping in the first place as you're only rendering one element. Just pass the element you need inside as a prop

Reply:
Ah so, you're saying it'll be better to pass the activeTab content directly as a prop to this Tab component?

Reply:
@W3stside Please review https://github.com/gnosis/dex-react/pull/1634 to address this.